server/cluster/cluster.go

```diff
@@ -938,7 +939,12 @@ func (c *RaftCluster) RemoveStore(storeID uint64) error {
 	log.Warn("store has been offline",
 		zap.Uint64("store-id", newStore.GetID()),
 		zap.String("store-address", newStore.GetAddress()))
-	return c.putStoreLocked(newStore)
+	err := c.putStoreLocked(newStore)
+	if err == nil {
+		// set the store limit to unlimit
+		c.coordinator.opController.SetStoreLimit(store.GetID(), float64(math.MaxInt64), storelimit.Auto, storelimit.RegionRemove)
```

Comment:
Set the limit mode to manual to avoid the value being changed according to the cluster load.

Reply:
But here is not set by manually, if we use manual, will it be confused? Or do you have a better idea? (I know it will conflict with the cluster load mechanism)

Reply:
We can regard it as manual as long as it is not adjusted by PD automatically. The unlimited value can regard as the default value of an offline store?